clang/test/OpenMP/amdgcn_target_codegen.cpp

```diff
@@ -9,7 +9,7 @@
 #define N 1000
 
 int test_amdgcn_target_tid_threads() {
-// CHECK-LABEL: define weak void @{{.*}}test_amdgcn_target_tid_threads
+// CHECK-LABEL: define weak amdgpu_kernel void @{{.*}}test_amdgcn_target_tid_threads
```

Comment:
This looks suspect  - I don't see a target region

Reply:
I misread this - the target region below creates a function with a name derived from the containing function